src/odemis/dataio/tiff.py

```diff
@@ -3306,3 +3332,290 @@ def _convert_openfibsem_to_odemis_metadata(metadata: str) -> dict:
         raise ValueError("ImageDescription tag contained data, but no OpenFIBSEM metadata found... falling back to previous metadata...")
 
     return md
+
+def reformat_ome_metadata(image_data: List[model.DataArray],
+                          filename: str,
+                          ) -> None:
+    """Save an odemis image with OME-XML format (2016-06)
+    :param image_data: (list of DataArray) image data
+    :param filename: (str) filename to save the image
+    """
+
+    import tifffile as tff
```

Comment:
Please don't make a separate function for OME-TIFF 201606. It should be possible to just tweak the current code to "upgrade" from "201206++" to 201606. 
Here, you're introducing new dependencies, which is extra work to package and test, and especially, it's not clear what we are loosing. Does it support all the "special" metadata like MD_EXTRA_SETTINGS, or the metadata from the SPARC? Especially, does it support pyramidal imaging? We should aim at having one function that does things really well, instead of 2 equivalent functions with both their drawbacks and advantages.